seed/urls/main.py

```diff
@@ -151,6 +153,43 @@
         name='delete_organization_buildings'
     ),
 
+    # timeseries data
```

Comment:
@rebrownlbnl @RDmitchell @pipermerriam  I haven't thought enough about this... but what do we want the API endpoints for the time series data? We should verify the items below because they will probably exist for quite some time.

My immediate reaction is that we should chomp the URL portion from the endpoints, as it is already a url.